packages/mobile/src/hooks/selectors.ts

```diff
@@ -0,0 +1,19 @@
+import { getAccountUser } from 'audius-client/src/common/store/account/selectors'
+import { getProfileUser } from 'audius-client/src/common/store/pages/profile/selectors'
+import { isEmpty } from 'lodash'
+
+import { useSelectorWeb } from './useSelectorWeb'
+
+export const useProfile = () => {
+  return useSelectorWeb(
+    getProfileUser,
+    (a, b) =>
```

Comment:
My only concern here is that these equality functions may no longer be necessary when we remove the web dependency, because reference equality will work correctly then

Reply:
ill def try the `isEqual` now and see how many re-renders that results in. i think these choices were less to do with selector performance, and more to do with number of re-renders, cause the root only needs user_id and a few other constants, and dont want it rerendering for things like track length increasing

Reply:
At the very least, maybe we could make a linear to track this and write
`TODO(AUD-1234): This selector is not necessary when common store is vendored in native`